### ISSUE TITLE
Possible to save the spoken text ?

### ISSUE THREAD

**Issue by abhi18av** [soimort/translate-shell#182]
Hello @soimort , 

I was wondering whether it's possible to save the audio from `bing` or `yandex` to an `mp3` file using translate-shell?

**Comment by soimort** [comment]
It's not implemented yet (#77). Try this workaround: (for `google` or `yandex`)

```
$ trans -player "wget -P '$PWD' -U 'Mozilla/5.0 (your preferred UA)'" -b 'hello world'
```

or (for `bing`)

```
$ trans -player "cp -t '$PWD'" -b 'hello world' /bing
```

**Comment by abhi18av** [comment]
Ah, I see. Thanks :)

Feel free to close the issue, since it's similar to the one filed before.
